src/main/java/com/jcabi/dynamodb/maven/plugin/Instances.java

```diff
@@ -141,21 +142,25 @@ private void shutdown() {
      */
     private static Process process(final File dist, final int port,
         final File home, final List<String> args) throws IOException {
-        return new ProcessBuilder().command(
-            new String[] {
-                new File(home, "bin/java").getAbsolutePath(),
-                new StringBuilder("-Djava.library.path=")
-                    .append(dist)
-                    .append(System.getProperty("path.separator"))
-                    .append(new File(dist, "DynamoDBLocal_lib"))
-                    .toString(),
-                "-jar",
-                "DynamoDBLocal.jar",
-                "--port",
-                Integer.toString(port),
-                StringUtils.join(args, " "),
-            }
-        ).directory(dist).redirectErrorStream(true).start();
+        final List<String> baseCommand = Arrays.asList(
+            new File(home, "bin/java").getAbsolutePath(),
+            new StringBuilder("-Djava.library.path=")
+                .append(dist)
+                .append(System.getProperty("path.separator"))
+                .append(new File(dist, "DynamoDBLocal_lib"))
+                .toString()
+            ,
+            "-jar",
+            "DynamoDBLocal.jar",
+            "--port",
+            Integer.toString(port)
+        );
+        final List<String> commandWithArgs =
```

Comment:
The result of `Arrays.asList` is an immutable list of an internal class.  My upcoming changes will use a direct `ArrayList` to encapsulate the base commands before adding the arguments.